src/core/public/chrome/chrome_service.tsx

```diff
@@ -127,7 +127,7 @@ export class ChromeService {
         )
       )
     );
-    this.isVisible$ = combineLatest(this.appHidden$, this.toggleHidden$).pipe(
+    this.isVisible$ = combineLatest([this.appHidden$, this.toggleHidden$]).pipe(
```

Comment:
Did it change any logic?

Reply:
No, changed it because the previous signature is deprecated.